revert part of inheritAll=false
https://bz.apache.org/bugzilla/show_bug.cgi?id=56988

otherwise deps for jcl build don't get set

--- tc7.0.x/build.xml~	2015-07-26 22:42:44.873164872 +0300
+++ tc7.0.x/build.xml	2015-07-26 22:53:44.930780044 +0300
@@ -1715,7 +1715,7 @@
 
     <ant antfile="${tomcat.extras}/logging/commons-logging-${commons-logging.version}-src/build2.xml"
          dir="${tomcat.extras}/logging/commons-logging-${commons-logging.version}-src"
-         inheritAll="false" target="compile" >
+         inheritAll="true" target="compile" >
       <property name="source.version" value="${logging.compile.source}" />
       <property name="target.version" value="${logging.compile.target}" />
     </ant>

